lib/gui/modules/drive-scanner.js

```diff
@@ -16,100 +16,150 @@
 
 'use strict';
 
-const Rx = require('rx');
 const os = require('os');
 const _ = require('lodash');
 const EventEmitter = require('events').EventEmitter;
 const drivelist = require('drivelist');
+const debug = require('debug')('etcher:drive-scanner');
 const settings = require('../models/settings');
 
-const DRIVE_SCANNER_INTERVAL_MS = 2000;
-const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 0;
-const emitter = new EventEmitter();
+const DRIVE_SCANNER_DELAY_MS = 2000;
+const DRIVE_SCANNER_FIRST_SCAN_DELAY_MS = 125;
```

Comment:
Would it be 'better' to not start the drive-scanner at all, until Etcher has finished loading / initialising itself?